sqldb/sqlc/migrations/000002_invoices.up.sql

```diff
@@ -0,0 +1,57 @@
+-- The invoices table contains all the information shared by all the invoice 
+-- types. 
+CREATE TABLE IF NOT EXISTS invoices (
+        id INTEGER PRIMARY KEY,
+
+        memo TEXT,
+        amount_msat BIGINT NOT NULL, 
+        cltv_delta INTEGER NOT NULL,
+        expiry INTEGER NOT NULL,
+
+        features BLOB NOT NULL,
+
+        payment_addr BLOB UNIQUE,
+        preimage BLOB UNIQUE,
+        hash BLOB UNIQUE NOT NULL,
+        
+        state SMALLINT NOT NULL,
+        
+        payment_request TEXT UNIQUE, 
+
+        last_settled_index INTEGER,
+        last_settled_at TIMESTAMP,
+        last_amount_paid_msat BIGINT NOT NULL,
+
+        is_amp BOOLEAN NOT NULL,
+        is_hodl BOOLEAN NOT NULL,
+        is_keysend BOOLEAN NOT NULL,
+
+        created_at TIMESTAMP NOT NULL
+);
+
+-- 
+CREATE TABLE IF NOT EXISTS settled_index (
+    id INTEGER PRIMARY KEY,
```

Comment:
Do we need this id?

Reply:
I added now a comment to make clear every field. This id is `settle_index`.